core/src/price_estimation/orderbook_based.rs

```diff
@@ -25,11 +24,7 @@ impl PriceSource for PricegraphEstimator {
             let (account_state, orders) =
                 self.orderbook_reader.get_auction_data(batch.into()).await?;
             let mut orderbook = Orderbook::from_elements(orders.iter().map(|order| {
-                order.to_element(U256(
-                    account_state
-                        .read_balance(order.sell_token, order.account_id)
-                        .0,
-                ))
+                order.to_element(account_state.read_balance(order.sell_token, order.account_id))
```

Comment:
So what type is it now then?

Reply:
It's still a U256 but now also defined in the same library and the same version as the one that the `Element` struct is expecting.